### ISSUE TITLE
Doesn't detect Quilt modloader

### ISSUE THREAD

**Issue by Slooshie1** [Mr-Technician/BorderlessMinecraft#37]
This program can detect fabric and vanilla instances just fine, as well as optifine instances (I have not tried iris instances) and make them borderless automatically when they launch. But quilt, a fork of fabric that is expected to one day replace fabric, isn't detected and the window is small when it is launched. I have to manually go borderless with the program.

**Comment by Mr-Technician** [comment]
You're still able to go borderless, correct? It just doesn't go borderless automatically when using quilt?

**Comment by Slooshie1** [comment]
Correct, I am able to go borderless manually by refreshing the instances
list and clicking go borderless

**Comment by Mr-Technician** [comment]
Ok, this is due to how the detection of processes works. It's something I would like to improve in the future but for now it's more of a "bad feature" than a bug.

**Comment by Felix14-v2** [comment]
> quilt, a fork of fabric that is expected to one day replace fabric

Haha, good luck.
Quilt was a community mistake, it already creates more problems than solutions.
